frontend/src/templates/DashboardTemplate.js

```diff
@@ -19,74 +18,89 @@ export function DashboardTemplate({}) {
     <>
       <TopNavigationLogged />
       <MainSectionDashboard>
-        <h1 style={{ textAlign: 'left', color: 'greenyellow' }}>Dashboard</h1>
-        <h2 style={{ textAlign: 'left', color: 'greenyellow' }}>
-          Nové tréninky
-          <NavLink to={route.newTraining('tt123')} className="pa3">
-            <div className="right-offset pt4">
-              <ButtonReverse>Nový trénink</ButtonReverse>
-            </div>
-          </NavLink>
-        </h2>
-        <main className="grid-container-left">
-          <div className="right-offset">
-            <ButtonReverse>
+        <div>
+          <div className="ph3">
+            <h1 className="f2 fw8 tracked">
+              Dashboard
+            </h1>
+              <h2 className="f3 fw5 tracked">
+                New Training
+              <NavLink to={route.newTraining('tt123')} className="pa3">
+                <a className="f4 tc link dim br-pill ph4 pv2 mb2 dib green bg-dark"
+                  href="#0">
+                  <div>Add new </div>
+                </a>
+              </NavLink>
+              <div className="right-offset"></div>
+              </h2>
+            <a className="f7 tc link dim br-pill ph4 pv2 mb2 ma2 dib green bg-dark"
```

Comment:
klasický anchor je asi spíš link? nebo to je temporary a bude se to ještě řešit?